Setup/shield-version.txt

```diff
@@ -1,5 +1,5 @@
-#Build Dev:Build_541 on 19/06/26
-SHIELD_VER=8.0.0.latest SHIELD_VER=Dev:Build_541
+#Build Dev:Build_542 on 19/06/30
+SHIELD_VER=8.0.0.latest SHIELD_VER=Dev:Build_542
```

Comment:
Please remove 'Dev'
should be Rel-19.06.542

Reply:
@BenyH this is a dev build